_posts/2020-04-05-hello.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Hello World!
+author: John
+---
+
+<span class="image right">
+    <img src="/images/blog/edmonton-skyline.jpg" alt="">
+    <small>
+        <a href="https://www.flickr.com/photos/iqremix/42276507674">Edmonton Summer Cityscape</a> by <a href="https://www.flickr.com/photos/iqremix/">IQRemix</a>, used under <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA 2.0</a>
```

Comment:
Not supporting Markdown in HTML is the one part of Markdown spec that drives me crazy. This looks so ugly, but 🤷

Reply:
I'm not going to block this PR, but I'm strongly against using HTML, even if it leads to richer layouts like this.

1. It breaks the blog list page's excerpt capability, which blocks the ability for me to add a link to our blog from the home page.
   ![image](https://user-images.githubusercontent.com/6334517/78949569-2e0e0380-7a89-11ea-830b-ce9d288a5734.png)
1. Subjectively, the image and its attribution don't respond well to mobile
   ![image](https://user-images.githubusercontent.com/6334517/78949628-539b0d00-7a89-11ea-98fd-893ee2455593.png)

It's not the end of the world, but I think I'll submit a mini-PR after the fact to just make this easier by making it an image with some of the first and bottom third of it cropped out.